mod/b_extended_profile/views/default/b_extended_profile/edit_opt-in.php

```diff
@@ -14,41 +14,13 @@
  * This view is inside a section wrapper as described in wrapper.php.
  */
 if (elgg_is_xhr) {
-	echo elgg_echo ( 'gcconnex_profile:opt:opt_in_access' );
-	
 	$user_guid = elgg_get_logged_in_user_guid();
 	$user = get_user ( $user_guid );
-	
-	$access_id = $user->optaccess;
-	$params = array (
-			'name' => "accesslevel['optaccess']",
-			'value' => $access_id,
-			'class' => 'gcconnex-opt-in-access' 
-	);
-	echo elgg_view ( 'input/access', $params );
-	
-	// Decides whether or not the checkbox should start checked.
-	/*if($user->opt_in_missions == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[0] = true;
-	}
-	if($user->opt_in_swap == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[1] = true;
-	}
-	if($user->opt_in_mentored == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[2] = true;
-	}
-	if($user->opt_in_mentoring == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[3] = true;
-	}
-	if($user->opt_in_shadowed == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[4] = true;
-	}
-	if($user->opt_in_shadowing == 'gcconnex_profile:opt:yes') {
-	    $opt_in_set[5] = true;
-	}*/
-    
-    
-                $opt_in_set = array($user->opt_in_missions, $user->opt_in_swap, $user->opt_in_mentored, $user->opt_in_mentoring, $user->opt_in_shadowed, $user->opt_in_shadowing, $user->opt_in_jobshare, $user->opt_in_pcSeek, $user->opt_in_pcCreate, $user->opt_in_ssSeek, $user->opt_in_ssCreate, $user->opt_in_rotation, $user->opt_in_assignSeek, $user->opt_in_assignCreate, $user->opt_in_deploySeek, $user->opt_in_deployCreate, $user->opt_in_missionCreate);
+
+	echo elgg_format_element('div',array('class'=> 'mrgn-bttm-sm mrgn-tp-sm alert alert-info'),elgg_echo('gcconnex_profile:optin:access'));
+
+
+	 $opt_in_set = array($user->opt_in_missions, $user->opt_in_swap, $user->opt_in_mentored, $user->opt_in_mentoring, $user->opt_in_shadowed, $user->opt_in_shadowing, $user->opt_in_jobshare, $user->opt_in_pcSeek, $user->opt_in_pcCreate, $user->opt_in_ssSeek, $user->opt_in_ssCreate, $user->opt_in_rotation, $user->opt_in_assignSeek, $user->opt_in_assignCreate, $user->opt_in_deploySeek, $user->opt_in_deployCreate, $user->opt_in_missionCreate);
```

Comment:
- there is a leading space after the first tab
- Might be easier to read if you split the array to multiple lines (I know it was already like this though :wink: )